lib/ruby_lsp/requests/support/common.rb

```diff
@@ -26,7 +26,7 @@ def range_from_syntax_tree_node(node)
           params(node: T.any(SyntaxTree::ConstPathRef, SyntaxTree::ConstRef, SyntaxTree::TopConstRef)).returns(String)
         end
         def full_constant_name(node)
-          name = +node.constant.value
+          name = +node.constant.value.dup
```

Comment:
I believe `dup` will return a unfrozen string, so the `+` shouldn't be necessary?